circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3CopierOptions.java

```diff
@@ -77,6 +87,7 @@ public String keyName() {
   }
 
   private final Map<String, Object> copierOptions;
+  public static final int USE_DEFAULT_MAX_THREAD_POOL = -1;
```

Comment:
perhaps default this to 10 (as per readme)  and remove the if check for -1 in the TransferManagerFactory. Just always set our own.